operator/src/main/resources/scripts/utils.sh

```diff
@@ -395,3 +395,26 @@ checkWebLogicVersion()
   fi
   return 0
 }
+
+
+#
+# getAdminUrl
+#   purpose: Get the ADMIN_URL used to connect to the admin server internally, e.g. when starting a managed server
+#   sample:
+#     ADMIN_URL=$(getAdminUrl)
+#
+function getAdminUrl() {
```

Comment:
maybe change name to 'getAdminServerUrl' to help make clear this function returns url specific to the admin server...

Reply:
Noted.  Will make the change.